app/views/users/edit.html.erb

```diff
@@ -21,25 +21,7 @@
   <% end %>
 
   <div class="crayons-layout crayons-layout--header-inside crayons-layout--2-cols crayons-layout--limited-l">
-    <div class="crayons-page-header">
-      <% if @tab == "organization" && @organizations.present? %>
-        <h1 class="crayons-title">
-            <% options = @organizations.pluck(:slug, :id).each do |arr| %>
-              <% arr[0] = "@#{arr[0]}" %>
-              <% arr[1] = "/settings/organization/#{arr[1]}" %>
-            <% end.to_h.merge(t("views.settings.new_org") => "/settings/organization/new") %>
-            <%= t("views.settings.for_html", name: tag.select(options_for_select(options, "/settings/organization/#{params[:org_id]}"), class: "ml-1 crayons-select lh-tight w-25 align-text-top w-max", onchange: "window.location.href=this.value")) %>
-            <%# For example: %>
-            <%# <option value="/settings/organization/1>@devteam</option> %>
-            <%# <option value="/settings/organization/new" selected>New Organization</option> %>
-        </h1>
-      <% else %>
-        <h1 class="crayons-title">
-          <%= t("views.settings.for_html", name: link_to("@#{@user.username.truncate(User::USERNAME_MAX_LENGTH)}", user_url(@user) || "#", class: "ml-1")) %>
-        </h1>
```

Comment:
@anujbhavsar96 just want to make sure that you're aware that this removes the "Settings for" heading on all /settings pages and that we're okay with that?

Reply:
Yes @Ridhwana - That title is unnecessary and repetitive